pkg/models/commandvalue.go

```diff
@@ -81,6 +85,11 @@ type CommandValue struct {
 	NumericValue []byte
 	// stringValue is a string value returned as a value by a ProtocolDriver instance.
 	stringValue string
+	// BinValue is a CBOR encoded binary value with a maximum
+	// capacity of 1MB, used to hold binary values returned
```

Comment:
Where is the 1MB max constraint enforced?

Reply:
Good catch, added @ [line 197](https://github.com/edgexfoundry/device-sdk-go/pull/232/files#diff-a2c6736ced964ce680cce6159abd2598R197)